theme_avantgarde/views/customizations.xml

```diff
@@ -1,60 +1,155 @@
     <xpath expr="//section" position="attributes">
-        <attribute name="class" add="s_parallax_no_overflow_hidden" remove="s_parallax_is_fixed s_parallax" separator=" "/>
+        <attribute name="class" add="s_parallax_no_overflow_hidden o_full_screen_height" remove="s_parallax_is_fixed s_parallax" separator=" "/>
+        <attribute name="data-oe-shape-data">{"shape":"web_editor/Origins/18","colors":{"c1":"#0B0D63"},"flip":["x"]}</attribute>
     </xpath>
     <xpath expr="//span[hasclass('s_parallax_bg')]" position="attributes">
         <attribute name="style" remove="background-position: 50% 0;" add="background-position: 50% 80%;" separator=";"/>
     </xpath>
+    <xpath expr="//div[hasclass('s_allow_columns')]" position="before">
+        <div class="o_we_shape o_web_editor_Origins_18" style="background-image: url(&quot;/web_editor/shape/web_editor/Origins/18.svg?c1=%230B0D63&amp;flip=x&quot;); background-position: 50% 50%;"/>
+    </xpath>
+    <xpath expr="//font" position="attributes">
+        <attribute name="class" add="text-o-color-1 text-break" separator=" "/>
+    </xpath>
+    <xpath expr="//font" position="replace" mode="inner">
+        We are Avantgarde.
+    </xpath>
+    <xpath expr="//p" position="replace" mode="inner">
+        We create architecture, landscapes, interiors, product design and graphic design.<br/>
+        Our methods allow us to explore a future that is equitable, data-driven, and green.
+    </xpath>
     <xpath expr="//a[hasclass('btn')]" position="attributes">
         <attribute name="class" add="btn-lg" separator=" "/>
     </xpath>
 </template>
 
-<!-- ======== BANNER ======== -->
- <template id="s_banner" inherit_id="website.s_banner" name="Avantgarde s_banner">
-    <xpath expr="//a[hasclass('btn')]" position="attributes">
-        <attribute name="class" add="btn-lg" separator=" "/>
+<!-- ======== PICTURE ======== -->
+ <template id="s_picture" inherit_id="website.s_picture" name="Avantgarde s_PICTUREcover">
+    <xpath expr="//section" position="attributes">
+        <attribute name="class" add="pt144 pb128" remove="pt48 pb24" separator=" "/>
+    </xpath>
+    <xpath expr="//h2" position="attributes">
+        <attribute name="style" add="text-align: left;" separator=" "/>
+    </xpath>
+    <xpath expr="//font" position="attributes">
+        <attribute name="class" add="text-break" separator=" "/>
+        <attribute name="style" add="font-weight: bolder;" separator=" "/>
+    </xpath>
+    <xpath expr="//font" position="replace" mode="inner">
+        Design Methodology
+    </xpath>
+    <xpath expr="//p" position="attributes">
+        <attribute name="style" add="text-align: left;" separator=" "/>
+    </xpath>
+    <xpath expr="//p" position="replace" mode="inner">
+        Our design methodology is rooted in our core values.<br/>
+        We combine a wide spectrum of technologies with a good portion of creativity.
+    </xpath>
+    <xpath expr="//img" position="attributes">
```

Comment:
Done 👍 

I didn't know we were no longer using those. 🤔

Reply:
Sure about that @Cocographique ?

Reply:
Indeed, the `data-file-name` attribute is still needed.
@karlosvalverde, you can put it back. Sorry 😅

Reply:
@Cocographique @qsm-odoo And also `data-original-mimetype`, right?
I'm putting them back 😉

Reply:
Yes, probably even more important :wink: Even though I agree this is kinda stupid we force you to put the same value everywhere instead of assuming it as default value in case there is a shape URL but... one step at a time :grimacing: